src/herder/HerderSCPDriver.cpp

```diff
@@ -319,18 +316,17 @@ HerderSCPDriver::validateValueHelper(uint64_t slotIndex, StellarValue const& b,
         return SCPDriver::kInvalidValue;
     }
 
-    if ((!nomination || lcl.ledgerVersion < 11) &&
-        b.ext.v() != STELLAR_VALUE_BASIC)
+    bool const expectSignedValue =
```

Comment:
let's add a commit before the CAP34 changes that *removes* the pre protocol 11 behavior entirely, that way we can have this commit focus on new behavior only

Reply:
> let's add a commit before the CAP34 changes that _removes_ the pre protocol 11 behavior entirely, that way we can have this commit focus on new behavior only

Will that induce any change in this commit besides allowing...

```
    bool const expectSignedValue =
        nomination ? (lcl.ledgerVersion >= 11)
                   : (compositeValueType() == STELLAR_VALUE_SIGNED);
```

...to be reduced to...

```
    bool const expectSignedValue =
        nomination || (compositeValueType() == STELLAR_VALUE_SIGNED);
```

...?  (We could also remove that `SECTION("protocol 10")` test that you mentioned below, but that was there already.)

What will removing pre-protocol-11 behavior entail?  I see at least dozens of comparisons of ledger versions `> 10` or `>= 11`, as well as with lower version numbers, throughout the code base; are you talking about removing all of that, and starting to ignore (?) messages from nodes speaking protocols lower than 11?

Reply:
Nico explained to me what he meant by removing pre-protocol-11 behavior, and I have now done that in a commit prior to the main CAP-34 commit.